myning/chapters/enter_graveyard.py

```diff
@@ -1,11 +1,10 @@
-from math import sqrt
+from functools import cache
```

Comment:
could probably remove this then since it's unused

Reply:
I'll get it in the next MR, coming soon.